src/highdicom/sr/content.py

```diff
@@ -103,6 +163,35 @@ def __init__(
             relationship_type=RelationshipTypeValues.INFERRED_FROM
         )
 
+    @classmethod
+    def from_source_image(
+            cls,
+            image_dataset: Dataset,
+            referenced_frame_numbers: Optional[Sequence[int]] = None
+        ) -> 'SourceImageForMeasurement':
```

Comment:
Let's start doing this consistently throughout the library and in compliance with PEP8:

```suggestion
        cls,
        image_dataset: Dataset,
        referenced_frame_numbers: Optional[Sequence[int]] = None
    ) -> 'SourceImageForMeasurement':
```